blog/13-05-2023-integrating-celo-payments-into-your-web-or-mobile-application/integrating-celo-payments-into-your-web-or-mobile-application.md

```diff
@@ -0,0 +1,483 @@
+
+To check the status of a transaction directly from your Flutter app, you need to interact with the Celo network or a Celo node.
+
+Here's a typical way you could handle it:
+
+Backend Server:
+
+1. Create a backend service (Node.js, Python, or any backend language you are comfortable with).
+2. The backend service would interact with the Celo network using a library like @celo/contractkit in Node.js.
+3. After the user initiates a transaction from the Flutter app, the backend service would continuously monitor the status of the transaction on the Celo network.
+4. Once the transaction is successful, the backend service would update the status in your application database.
+
+Flutter App:
+
+1. The Flutter app would send a request to initiate the transaction and then periodically send requests to the backend service to check the status of the transaction. Or you can make it such that the user clicks on a button called `confirm transaction` after making a payment.
+2. Once the backend service responds that the transaction is successful, the Flutter app would update the UI to reflect the transaction's success.
+
+## Conclusion​
+
+In this tutorial, we've taken a journey through the process of integrating Celo payments into both Flutter mobile and React web applications. We started from scratch, setting up the necessary environments, installing the required dependencies, and understanding how to connect to the Celo network. We then moved on to implement payment request forms, handle payment confirmations, and create user-friendly interfaces to interact with the payment systems.
+
+In the mobile application development using Flutter, we learned about deep linking, a technique that allows applications to communicate with each other, facilitating the payment process. On the other hand, in the React web application, we made use of the Web3 library to interface directly with a Celo wallet like MetaMask, allowing for seamless payment transactions on the web.
+
+Furthermore, we delved into understanding the structure and flow of transactions in the Celo network, from the request to the confirmation. This knowledge is crucial in building robust and reliable payment systems.
+
+Lastly, it's important to test thoroughly and ensure the security of your applications, particularly when dealing with financial transactions. Always stay up-to-date with best practices and updates from Celo and other relevant technology providers.
+
+By integrating Celo payments into your applications, you're not only providing a valuable service to your users, but also contributing to a more inclusive and accessible global financial system. Happy coding!
+
+## Next Steps​
+
+To continue learning and building on this foundation, consider exploring the following topics:
+
```

Comment:
Can you provide links to back these topics, or something nearest to the topic ideas, preferably on the cell sage, but feel free to go wild, just make sure its relevant and related

Reply:
Done

Reply:
Awesome 🥂